tests/external_executor/executors/logging_app.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+import grpc
+import json
+import urllib
+
+from loguru import logger as LOG
+
+# pylint: disable=import-error
+import kv_pb2 as KV
+
+# pylint: disable=import-error
+import http_pb2 as HTTP
+
+# pylint: disable=import-error
+import kv_pb2_grpc as Service
+
+# pylint: disable=no-name-in-module
+from google.protobuf.empty_pb2 import Empty as Empty
+
+
+class LoggingExecutor:
+    def __init__(self, ccf_node, credentials):
+        self.ccf_node = ccf_node
+        self.credentials = credentials
+
+    def do_post(self, kv_stub, table, request, response):
+        body = json.loads(request.body)
+        kv_stub.Put(
+            KV.KVKeyValue(
+                table=table,
+                key=body["id"].to_bytes(8, "big"),
+                value=body["msg"].encode("utf-8"),
+            )
+        )
+        response.status_code = HTTP.HttpStatusCode.OK
+        header = response.headers.add()
+        header.field = "content-type"
+        header.value = "application/json"
+        response.body = json.dumps(True).encode("utf-8")
```

Comment:
Beyond the scope of this PR: are there any plans to look at how this would integrate with existing HTTP frameworks (e.g. Flask) and if so, provide a sample app?

Reply:
No concrete plans, but I think it's something we'll look at when building more sample apps.